### DISCUSSION TITLE
V3.5.2 - How to enable the doc window vertical scroll bar in Single page non-continuous?

### DISCUSSION THREAD

**Discussion by chrisjj** [sumatrapdfreader/sumatrapdf#4641] [Q&A]
How may I enable the doc window vertical scroll bar for page-to-page when View has "* Single page" and unticked Show pages continuously?

Thanks.

**Comment by GitHubRulesOK** [comment]
long time open issue and heated discussion https://github.com/sumatrapdfreader/sumatrapdf/issues/191
but there is no need for scrollbars in MuPDF core viewer for single pages.
However are needed by SumatraPDF for **C**ontinuous scrolling so will  appear when you press C or toggle on when not in fit ONE page

**Comment by chrisjj** [comment]
> but there is no need for scrollbars in MuPDF core viewer for single pages.

Well, I am using SumatraPDF. And I do need it.

I guess the answer is "You cannot"?
